### PR TITLE
plan: call the prepareStats but not use it yet.

### PR DESCRIPTION
**Pull request by hanfei1991**
This is the a pre-step of stats propagation, just call it and increase the coverage of test to 79.7%

### CODE REVIEW

**Comment by hanfei1991** [comment]
@zimulala @winoros @lamxTyler PTAL

File: plan/stats.go [hunk 1]
```diff
@@ -26,54 +26,70 @@ type statsProfile struct {
 	cardinality []float64
 }
 
+func (p *basePhysicalPlan) statsProfile() *statsProfile {
+	return p.basePlan.profile
+}
+
 // Sort will call this.
```
Comment (lamxTyler) [inline plan/stats.go]: The comment is out-dated?

Comment (hanfei1991) [inline plan/stats.go]: Well, this comment is useless.

File: plan/stats.go [hunk 2]
```diff
@@ -186,12 +189,12 @@ func (p *LogicalAggregation) statsProfile() *statsProfile {
 // N(s) stands for the number of rows in relation s. V(s.key) means the cardinality of join key in s.
 // This is a quite simple strategy: We assume every bucket of relation which will participate join has the same number of rows, and apply cross join for
 // every matched bucket.
-func (p *LogicalJoin) statsProfile() *statsProfile {
+func (p *LogicalJoin) prepareStatsProfile() *statsProfile {
 	if p.profile != nil {
 		return p.profile
 	}
```
Comment (lamxTyler) [inline plan/stats.go]: Why only the LogicalJoin keeps this test?

Comment (hanfei1991) [inline plan/stats.go]: my fault

**Comment by lamxTyler** [comment]
LGTM

**Pull request merged**